static/img/shared/white-paper-dark.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't see anything referencing this file in this PR.

Reply:
It's used by libra-developer-plugins. Some image have to go in the main repo folder (libra-website, lips), so they go in the img/shared folder

Reply:
Weird, no way around that?

Reply:
I'll look into it